docs/README.md

```diff
@@ -107,6 +107,12 @@ _DearPyGui_ includes several tools to help developers with _DearPyGui_ app devel
 The actual builds of _DearPyGui_ are created through github actions however, to develop and test locally, follow the instructions below. 
 These are not well maintained at the moment and could use some work.
 
+---
```

Comment:
This should be moved above **Building**

Also add links to the reddit and discord, then it will be good to go!

Reply:
Ok, will do it from atom